### DISCUSSION TITLE
Request for help regarding the definition of floppy drive pins

### DISCUSSION THREAD

**Discussion by bodong001** [keirf/flashfloppy#943] [General]
I changed a 34 pin floppy drive simulator to a 26 pin ribbon cable connection, but a strange problem occurred with the conversion board I purchased in China: the floppy drive cannot be recognized on my oscilloscope. I modified a lead to recognize it on my oscilloscope. It only reads the first image file each time, and after switching to other image files, it still reads the content of the first read image. It can be driven by an external USB on a personal computer. It also only reads the first image file at a time. So I need pin definitions for 34 pin and 26 pin floppy drives to determine the style. I hope everyone can help me provide pin definitions. Thank you!

**Comment by keirf** [comment]
https://allpinouts.org/pinouts/connectors/data_storage/35-inch-micro-floppy-disk-drive-26-pin/

https://old.pinouts.ru/HD/InternalDisk_pinout.shtml (see: Floppy drive pinout, Shugart interface)

Bear in mind that whether JC jumper is installed, or not, affects pins 2 and 34:
* Installed: 34 = DSKCHG, 2 = n/a
* Missing: 34 = READY, 2 = DSKCHG

Depends on your adapter, but you probably *don't* want JC.

**Comment by bodong001** [comment]
Thank you very much. I should check where the problem lies with my conversion board according to the foot definition. There will be good news. Thank you for your hard work!   @ keirf

**Comment by bodong001** [comment]
hi~keirf !
My problem has been solved. It was caused by me forgetting to close the SO jumper. In addition, according to your prompt, I modified pin 2 of pin 34. Now the oscilloscope is working very happily! Thank you again!
